posts/how-to-create-a-quick-react-and-angular-app-with-jhipster-lite.md

```diff
@@ -0,0 +1,457 @@
+---
+authors:
+- Dennis Sharpe
+tags:
+- JHipster
+- JHipster Lite
+- React
+- Angular
+- Hexagonal Architecture
+date: 2022-12-13T12:21:50.000Z
+title: "How to Create a Quick React and Angular App with JHipster Lite"
+image: https://raw.githubusercontent.com/ippontech/blog-usa/master/images/2022/12/jhipster_lite_logo.png
+---
+# What is JHipster Lite?
+
+JHipster Lite is an application generator that generates step-by-step only what you need for a modern web application and/or microservice architecture. As you might imagine from the name, JHipster Lite is a different approach to application generation than its predecessor, [JHipster](https://www.jhipster.tech/).
+
+Unlike JHipster, a front-end only or back-end only application can be generated. (JHipster does support just generating microservices) No questions are asked during generation; you decide which components you want as part of the generation process.
+
+![JHipster Lite Patches screen](https://raw.githubusercontent.com/ippontech/blog-usa/master/images/2022/12/jhipster_lite_patches.png)
+
+![JHipster Lite Spring Boot screen](https://raw.githubusercontent.com/ippontech/blog-usa/master/images/2022/12/jhipster_lite_spring_boot.png)
+
+![JHipster Lite Angular screen](https://raw.githubusercontent.com/ippontech/blog-usa/master/images/2022/12/jhipster_lite_angular.png)
+
+JHipster Lite does not currently generate entities or CRUD pages for the front-end. The idea is to set you up with a relatively blank slate with some initial structure. This provides a good starting pattern that can be followed as the application grows.
+
+JHipster Lite also generates code that follows a [Hexagonal Architecture](https://alistair.cockburn.us/hexagonal-architecture/). This helps with designing a system around the business intent.
+
+# Let’s Build a React App
+
+## Run JHipster Lite locally, if you want
+
+You can certainly go straight to the [JHipster Lite website](https://lite.jhipster.tech/landscape) and start generating there. But you can also run it locally if you have Docker. (or in my case, Rancher)
+
```

Comment:
I suggest linking to [Docker](https://www.docker.com) and/or linking to how to install docker if the reader don't already have it installed (https://docs.docker.com/get-docker)